site/en/docs/extensions/support-feedback/find-a-bug/index.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: 'layouts/doc-post.njk'
+title: Find and follow a bug
+description: 'TBD'
+date: 2023-10-11
+tags:
+  - extensions
+---
+
+## Find and follow a bug
+
+Before filing a bug or submitting a feature request for extensions, you should check whether either has already been reported by checking open Chromium issues. These issues are kept in [the Chromium issue tracker](https://crbug.com). This database lets you search for and follow any issue that's been reported for Chromium.
+
+1.  Open the [the issue tracker](https://crbug.com) and use the search box at the top to verify whether the issue has already been reported. Most extension issues are filed under `component=Platform>Extensions`. For example, a search for `component=Platform>Extensions Type=Bug chrome.scripting.executeScript` gives you [this list of results](https://bugs.chromium.org/p/chromium/issues/list?can=2&q=component%3DPlatform>Extensions+Type%3DBug+chrome.scripting.executeScript).
+1.  If you find a ticket that matches your bug or feature request, click the star icon to express your interest and to be notified when the bug is updated.
+1.  If you're not signed in, you will be prompted to do so. If you do not have an account in the issue tracker you may create one by clicking **Create account** on the sign-in prompt and following the on-screen instructions.
+
+Please do not respond to any bugs or feature requests by saying "me too" or by asking, "when will this be fixed?". However, if you have valuable information, such as a better test case or a suggested fix, please add a comment.
```

Comment:
Slight tone tweak (if a bug has been inactive for a long time, and it's important, I want people to feel like they do have a way of contacting us):

```suggestion
Please do not respond to any bugs or feature requests by saying "me too", and prefer using the [Google group](https://groups.google.com/a/chromium.org/group/chromium-extensions/topics) for updates on when a bug will be fixed. However, if you have valuable information, such as a better test case or a suggested fix, please add a comment.
```